compose/docker-compose.yaml

```diff
@@ -47,7 +47,8 @@ services:
       - "5000"
     ports:
       # export locally as 5001 to allow running a debug service at 5000
```

Comment:
This comment needs to be changed - also needs to add a note that 5011 -> 5001 is for the wasm filter, and above we need to _expose_ port `5001` as well.